### PR TITLE
Update use-dkim-to-validate-outbound-email.md

### PR DESCRIPTION
**Pull request by puneethmeister**
Currently we do not auto rotate, hence the option a is not valid "a) let the Microsoft 365 service rotate the selector and upgrade to 2048-bitness within the next 6 months"

### CODE REVIEW

**Comment by puneethmeister** [comment]
cc : @MSFTTracyP

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

File: microsoft-365/security/office-365-security/use-dkim-to-validate-outbound-email.md [hunk 1]
```diff
@@ -121,7 +121,7 @@ Get-DkimSigningConfig -Identity <Domain for which the configuration was set> | F
 > [!TIP]
 > This new 2048-bit key takes effect on the RotateOnDate, and will send emails with the 1024-bit key in the interim. After four days, you can test again with the 2048-bit key (that is, once the rotation takes effect to the second selector).
 
-If you want to rotate to the second selector, your options are a) let the Microsoft 365 service rotate the selector and upgrade to 2048-bitness within the next 6 months, or b) after 4 days and confirming that 2048-bitness is in use, manually rotate the second selector key by using the appropriate cmdlet listed above.
+If you want to rotate to the second selector, After 4 days and confirming that 2048-bitness is in use, manually rotate the second selector key by using the appropriate cmdlet listed above.
```
Comment (JohanFreelancer9) [inline microsoft-365/security/office-365-security/use-dkim-to-validate-outbound-email.md]:
```suggestion
If you want to rotate to the second selector, after four days and confirming that 2048-bitness is in use, manually rotate the second selector key by using the appropriate cmdlet listed above.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @puneethmeister, kindly commit the minor edit. Many thanks.

**Comment by yogkumgit** [comment]
@puneethmeister Gentle reminder

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@MSFTTracyP Please merge this PR. Thanks

**Comment by puneethmeister** [comment]
@MSFTTracyP Gentle reminder

**Review by MSFTTracyP — COMMENTED** [review]
Thanks

**Pull request merged**